control-plane/api-gateway/binding/binder.go

```diff
@@ -238,6 +243,19 @@ func (b *Binder) Snapshot() *Snapshot {
 			b.config.Gateway.Status = status
 			snapshot.Kubernetes.StatusUpdates.Add(&b.config.Gateway)
 		}
+
+		for idx, policy := range b.config.Policies {
+			policy := policy
+
+			var policyStatus v1alpha1.GatewayPolicyStatus
+
+			policyStatus.Conditions = policyValidation.Conditions(policy.Generation, idx)
+			// only mark the policy as needing a status update if there's a diff with it's old status
```

Comment:
"its" is possessive, "it's" is "it is".

```suggestion
			// only mark the policy as needing a status update if there's a diff with its old status
```

Reply:
words are hard